### PR TITLE
Add functions

### PR DESCRIPTION
**Pull request by waryiar**
Implementing 2 functions for RSSchool CodeJam-3!

### CODE REVIEW

File: src/sumOfOther.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module.exports = function sumOfOther(arr) {
+  const newArr = [];
+  for (let i = 0; i < arr.length; i + 1) {
```
Comment (andrei-shelenhouski) [inline src/sumOfOther.js:3]: @warryr This is an endless loop.

File: src/sumOfOther.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    const temp = [];
+    for (let j = 0; j < arr.length; j + 1) {
```
Comment (andrei-shelenhouski) [inline src/sumOfOther.js:5]: @warryr This is an endless loop.

File: src/make.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+module.exports = function make(...args) {
```
Comment (andrei-shelenhouski) [inline src/make.js:1]: @warryr we need to discuss this function. It seems not all cases are considered.

**Review by andrei-shelenhouski — COMMENTED** [review]
@warryr 
sumOfOther = 10
make = 20
recursion = 20 * 0 
git = 20
package.json/linter = 20
tests = 0
all creteria = 0

*70*
=====
FYI: @OlgaMardvilko

**Comment by OlgaMardvilko** [comment]
@manuminsk, @warryr  
 
Good job and good score!

**Pull request merged**